src/react_native_web/events.cljs

```diff
@@ -56,6 +56,11 @@
     (seqable? o) (empty? o)
     :else (nil? o)))
 
+(re-frame/reg-event-fx
+  :fiddle/set-app-db
+  (fn [{:keys [db]} [_ {id :hook-id} m]]
+    {:db (assoc-in db [:extensions/store id] m)}))
```

Comment:
same here for id as in subscription, and also we need to read this m data, currently you assoc string and it brakes db

Reply:
Ooops

Reply:
BTW why hook-id, not extension-id?

Reply:
this is the answer why it works in deployed version :) it should be `{id :id}` , but because we have only one store at the time it works with `nil` as well